apiserver/common/machine_test.go

```diff
@@ -114,26 +127,32 @@ func (s *machineSuite) TestMachineHardwareInfo(c *gc.C) {
 			"3": {life: state.Dying},
 		},
 	}
-	info, err := common.ModelMachineInfo(&st)
+	s.machineService.EXPECT().GetMachineUUID(gomock.Any(), machine.Name("1")).Return("uuid-1", nil)
+	s.machineService.EXPECT().InstanceID(gomock.Any(), "uuid-1").Return("123", nil)
+	s.machineService.EXPECT().GetMachineUUID(gomock.Any(), machine.Name("2")).Return("uuid-2", nil)
+	s.machineService.EXPECT().InstanceID(gomock.Any(), "uuid-2").Return("456", nil)
+	info, err := common.ModelMachineInfo(context.Background(), &st, s.machineService)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(info, jc.DeepEquals, []params.ModelMachineInfo{
 		{
-			Id:          "1",
-			DisplayName: "",
+			Id:         "1",
+			InstanceId: "123",
 			Hardware: &params.MachineHardware{
 				Arch:     &amd64,
 				Mem:      &gig,
 				Cores:    &one,
 				CpuPower: &one,
 			},
 		}, {
-			Id:          "2",
-			DisplayName: "",
+			Id:         "2",
+			InstanceId: "456",
```

Comment:
What's going on with DisplayName and the fact that we now return the InstanceId?

Reply:
Display name was added.